sematic/BUILD

```diff
@@ -29,8 +29,8 @@ sematic_py_lib(
         "gevent",
```

Comment:
Please add an entry to the changelog mentioning that an internal concept renaming took place.